qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/SnapshotWithExportersTest.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.clustering;
+
+import static io.camunda.zeebe.test.StableValuePredicate.hasStableValue;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.qa.util.actuator.PartitionsActuator;
+import io.camunda.zeebe.qa.util.testcontainers.ZeebeTestContainerDefaults;
+import io.camunda.zeebe.snapshots.impl.FileBasedSnapshotId;
+import io.camunda.zeebe.test.util.socket.SocketUtil;
+import io.zeebe.containers.ZeebeContainer;
+import io.zeebe.containers.ZeebeVolume;
+import io.zeebe.containers.exporter.DebugReceiver;
+import java.time.Duration;
+import java.util.stream.IntStream;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+
+final class SnapshotWithExportersTest {
+
+  @Test
+  void shouldIncludeExportedPositionInSnapshot() {
+    // given
+    try (final var debugReceiver =
+        new DebugReceiver((record) -> {}, SocketUtil.getNextAddress(), true).start()) {
+      try (final var zeebe =
+          new ZeebeContainer(ZeebeTestContainerDefaults.defaultTestImage())
+              .withDebugExporter(debugReceiver.serverAddress().getPort())) {
+        zeebe.start();
+
+        final var partitions = PartitionsActuator.of(zeebe);
+
+        try (final var client =
+            ZeebeClient.newClientBuilder()
+                .gatewayAddress(zeebe.getExternalGatewayAddress())
+                .usePlaintext()
+                .build()) {
+
+          publishMessages(client);
+          final var exporterPosition =
+              Awaitility.await("Exported position is stable")
+                  .atMost(Duration.ofSeconds(30))
+                  .during(Duration.ofSeconds(5))
+                  .until(() -> partitions.query().get(1).exportedPosition(), hasStableValue());
+          assertThat(exporterPosition).isPositive();
+
+          // when
+          partitions.takeSnapshot();
+
+          // then
+          final var exportedPositionInSnapshot =
+              Awaitility.await("Snapshot is taken")
+                  .atMost(Duration.ofSeconds(60))
+                  .during(Duration.ofSeconds(5))
+                  .until(
+                      () ->
+                          FileBasedSnapshotId.ofFileName(partitions.query().get(1).snapshotId())
+                              .orElseThrow()
+                              .getExportedPosition(),
+                      hasStableValue());
```

Comment:
:question: I don't understand why we wait for the exporter position of the snapshot to stabilize. Can't we take snapshots in a controlled way? Maybe with a ridiculously high interval? So we essentially have a high interval, then trigger a snapshot, and when we have one expect the exporter position to be the expected position?

Reply:
Oh yeah, that looks a bit confusing. I think at first this was just waiting for a snapshot to become available but then I merged extracting the exporter position in there and now it's misleading. 

> Can't we take snapshots in a controlled way? Maybe with a ridiculously high interval?

That's basically what we are doing already: snapshots are taken via the partitions actuator. The default snapshot interval should be high enough to not interfere with the tests here.